src/main/kotlin/no/nav/syfo/client/HttpClientCommon.kt

```diff
@@ -2,11 +2,16 @@ package no.nav.syfo.client
 
 import io.ktor.client.*
 import io.ktor.client.engine.cio.*
+import io.ktor.client.features.*
 import io.ktor.client.features.json.*
 import no.nav.syfo.util.configuredJacksonMapper
 
 fun httpClientDefault() = HttpClient(CIO) {
     install(JsonFeature) {
         serializer = JacksonSerializer(configuredJacksonMapper())
     }
+    install(HttpTimeout) {
+        requestTimeoutMillis = 120000
+        connectTimeoutMillis = 60000
```

Comment:
Er ikke 1-2 minutter veldig lenge?
Normalt sett burde ikke kallene våre ta mer enn maks et par sekunder.

Reply:
Jo, disse verdiene er kanskje litt høye. Men tenkte å se om det hadde noe å si for de 500-feilene som dukker opp flere ganger per dag. Er ikke helt sikker, men tror default er 15s.

Vi kan fjerne dette eller justere verdiene ned senere når vi har funnet ut mer.